packages/app/src/app/pages/common/ZeitIntegration/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+
+import ZeitLogo from 'app/components/ZeitLogo';
+import Integration from 'app/components/Integration';
+import { useStore, useSignals } from 'app/store';
+
+function ZeitIntegration({ small }: { small: boolean }) {
+  const { user, isLoadingZeit } = useStore();
+  const { signInZeitClicked, signOutZeitClicked } = useSignals();
+
+  return (
+    <Integration
+      name="ZEIT"
+      small={small}
+      color="black"
+      description="Deployments"
+      Icon={ZeitLogo}
+      userInfo={user.integrations.zeit}
+      signIn={() => signInZeitClicked()}
```

Comment:
@SaraVieira This can just be

```suggestion
      signIn={signInZeitClicked}
```